src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -93,6 +100,187 @@ function TimePeriodSelector(props) {
+
+  async function handleChange(value) {
+    if (value) {
+      await setBehavioralResponses();
+    } else {
+      await removeBehavioralResponses();
+    }
+
+    setIsChecked(value);
+  }
+
+  async function setBehavioralResponses() {
+    // Save the previous reform's label in case user reverts
+    prevLabelRef.current = policy.reform.label;
+    const policyToStack = behavioralResponseReforms[metadata.countryId];
+
+    // Fetch policy to stack
+    let newPolicyData = {
+      ...policy.reform.data,
+      ...policyToStack,
+    };
+
+    // Create new policy with those parameters and emit to back end, receiving back ID
+    const newPolicyRes = await getNewPolicyId(
+      metadata.countryId,
+      newPolicyData,
+    );
+    let newSearch = copySearchParams(searchParams);
+    newSearch.set("reform", newPolicyRes.policy_id);
+    setSearchParams(newSearch);
+  }
+
+  async function removeBehavioralResponses() {
+    const policyToStack = behavioralResponseReforms[metadata.countryId];
+
+    // Fetch policy to stack
+    let newPolicyData = {
+      ...policy.reform.data,
+    };
+
+    for (const key of Object.keys(policyToStack)) {
+      delete newPolicyData[key];
+    }
+
+    let newPolicyLabel = null;
+    // If there's a previous label, revert to it
+    if (prevLabelRef.current) {
+      newPolicyLabel = prevLabelRef.current;
+    }
+
+    // Current law is represented by an empty object;
+    // if the new policy is empty, remove the reform query parameter
+    if (Object.keys(newPolicyData).length === 0) {
+      let newSearch = copySearchParams(searchParams);
+      newSearch.delete("reform");
+      setSearchParams(newSearch);
+      return;
+    }
+
+    // Create new policy with those parameters and emit to back end, receiving back ID
+    const newPolicyRes = await getNewPolicyId(
+      metadata.countryId,
+      newPolicyData,
+      newPolicyLabel,
+    );
+    let newSearch = copySearchParams(searchParams);
+    newSearch.set("reform", newPolicyRes.policy_id);
+    setSearchParams(newSearch);
+  }
+
+  // Don't show behavior response toggle for countries without
+  // policies to set them
+  if (!Object.keys(behavioralResponseReforms).includes(countryId)) {
+    return null;
+  }
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "flex-start",
+        alignItems: "center",
+        gap: "10px",
+      }}
+    >
+      <Switch
+        checked={isChecked}
+        size={displayCategory !== "mobile" && "small"}
+        onChange={handleChange}
+      />
+      <p
+        style={{
+          margin: 0,
+          fontSize: displayCategory !== "mobile" && "0.95em",
+        }}
+      >
+        Apply {spellsBehaviour ? "behavioural" : "behavioral"} responses
+      </p>
+      <Tooltip
+        placement="topRight"
+        title={`When selected, use dynamic ${spellsBehaviour ? "behavioural" : "behavioral"} responses in simulations`}
```

Comment:
Can you also link to this here: https://www.cbo.gov/sites/default/files/112th-congress-2011-2012/reports/43674-laborsupplyfiscalpolicy.pdf#page=4

It looks like this doesn't currently include capital gains responses so I want to be clear on that.
```suggestion
        title={`When selected, simulations adjust earnings using the US Congressional Budget Office's ${spellsBehaviour ? "behavioural" : "behavioral"} income and substitution elasticities.`}
```